Zend/zend_attributes.h

```diff
@@ -92,6 +93,8 @@ ZEND_API zend_attribute *zend_add_attribute(
 
 uint32_t zend_attribute_attribute_get_flags(zend_attribute *attr, zend_class_entry *scope);
 
+ZEND_API ZEND_COLD zend_result ZEND_FASTCALL zend_attribute_get_nonpublic_suffix(HashTable *attributes, zend_string **message_suffix);
```

Comment:
Does it actually need to be a fastcall? As far as I saw, it's not on any hotpath

Reply:
`ZEND_FASTCALL` changes the calling convention, rather than improving performance. It should be present only on functions called from the JIT.

Reply:
OK, I thought that passing arguments in registers is useful for improving performance.

Reply:
`ZEND_FASTCALL` doesn't have an effect on x86-64 anymore, where register arg passing has become the default. But yes, I suppose `ZEND_FASTCALL` could still be faster on 32-bit.